src/github/branch.test.ts

```diff
@@ -43,15 +43,9 @@ describe("Branch Webhook", () => {
 		await sqsQueues.branch.purgeQueue();
 	});
 
-	describe.each([true, false])("Create Branch - New GH Client feature flag is '%s'", (useNewGithubClient) => {
+	describe("Create Branch - New GH Client feature flag is '%s'", () => {
```

Comment:
Don't need the description with %s as you don't have a parameter anymore.

Reply:
Fixed